### ISSUE TITLE
Adding the Document suite into Foundry

### ISSUE THREAD

**Issue by patmifsud** [Trunkplatform/lpma#38]
Hi Gnun, Harut

We'd like to add a couple of static pages to Foundry where users can download some zip files. 

Here are the wireframes for the site: 
https://www.figma.com/file/DGKZMOKhrIF6G70yPYo3V6Et/Foundry-V1.5-new-features

Here is the page hierarchy and content
https://docs.google.com/spreadsheets/d/1E4qY3MRqryoYjh4kPMl4bvZz6z8fwftGKroMb_7TCOc/edit#gid=0

There are 15 new pages in total. The pages are coloured orange in the hierarchy spreadsheet.

NOTE: if the urls are now foundry.lpma.com/ instead of lpma.com/foundry please update these urls to make sense with that naming scheme

**Comment by patmifsud** [comment]
Hi Gnun, this has been updated with a more complete brief, let me know if any further questions

**Comment by gnunua** [comment]
Hey @patmifsud,-  Here are questions which we would like to clarify before starting implementation:

---
- 1 - `Remove username, just welcome back`
    after auth implementation, we can get the username, so I assume it can remain. Like `Welcome back, gnun`. Should I remove this functionality?
http://prntscr.com/iusu3d
---
- 2 - Document Suite Card on foundry main page
http://prntscr.com/iusup0
I think we also should have `document suite` drop-down menu item, like `building a career` and `building your business`
http://prntscr.com/iusvvu
---
- 3 - There are 3 cards in document suit page http://prntscr.com/iute8o
 There is mentioned to **IGNORE FOR V1** card for New Zeland
Does this mean the content should not be filled in Contentful CMS for New Zealand version but the corresponding functionality should still remain?
---
- 4 - Are all cards ( for Australia, USA, and New Zealand ) visible to all visitors? For example, can LPMA members from the USA see Document Suite Card for Australia and New Zealand? Or should we somehow manage Documents suite displaying based on country? If yes, please provide the redirection rules.
---
- 5  - `Back to Foundry` Button http://prntscr.com/iutoby
Where should this button navigate to, one step back or to the main page of the foundry (/foundry) 
---
- 6 - In Level 5 `Client handbooks` and `Tenant handbooks`, the resources links do not exist http://prntscr.com/iuu8z9.
Should those be replaced by something else?
---
- 7  - Not clear why those links are marked in different color https://prnt.sc/iuu9ie
http://i.prntscr.com/eSLFhSOBRACSIq2fOGQeAw.png
---
- 8  - I see `Letters, checklists and forms` has 3 links 1) `All trust accounting systems`, 2) `Resources for Console nad ` and 3) `Resources for REST`. Each of them has the corresponding downloadable link. http://prntscr.com/iuudz6. So, can members download directly from both level-4 and level-5? If yes, in what way should users navigate from level-4 to level-5?
http://i.prntscr.com/35DUhkY4RJqT7TQFpw5hsg.png
---
- 9  -`Letters, checklists and forms` have 3 sub cards. What about `Client newsletter articles`, `Client handbooks` and `Tenant handbooks` 
---

**Comment by patmifsud** [comment]
1- Please remove, based on feedback
2- Great point, agreed
3- Yes please, it might be ready to implement soon
4- none of this is region specific right now, available on all sites
5-  other than on the Level 5 pages where noted, a user will never have any reason to navigate one step back to download additional resources. Their next step in user flow would be to return back to foundry. 
6- This will be linked up by the time the site is live, thanks for checking
7- Visual hierarchy. Not high importance.
8-  we want to be able to link directly to the states page when talking to our customers. May as well link the heading through for now, but agree it doesn't make sense out of context. 
9- no sub cards, it has a download icon and will just link through to downloading of a file

Thanks!

**Comment by patmifsud** [comment]
Thanks Gnun! Some quick feedback -

1. It needs some visual polish still, spacing, font sizes and layout is different to the mocks. Is there anything I can provide to help with this?  You should be able to get information about font sizes, colors and spacing from Figma. You might need to be invited or be given edit access if you are having trouble finding this info - let me know. 
2. USA links appear to be broken

Otherwise all good, thanks

**Comment by gnunua** [comment]
Hey @patmifsud. 
I have noticed you have changed UI and layout in figma. Please let me know in the case when you updating the UI. 
- 1 - There is the new tab called `Research Tab`.  http://prntscr.com/j03zru.
- 2 - I did not find descriptions of that new card. Please provide more detailed information.
For now, not clear which part to remove from `Welcome back`. You have been mentioned `please remove based on feedback`, but not clear which feedback you meant.
- 3 -  There is a new section with video http://prntscr.com/j042lg. with no description. Please provide video URL. 
- 4 - These parts disappear from Foundry page main on new UI http://prntscr.com/j04419. Should I remove appropriate functionality and content from Contentful?
- 5 - there is FB like functionality http://prntscr.com/j04693. Please provide a more information such as appId, URL,  etc 
- 6 -  I have checked USA links works, could you please provide info how do you reproduce it?
- 7 -  There is new button http://prntscr.com/j04ce8. Where to link? 
- 8  - http://prntscr.com/j04ce8, Could please clarify this feature?

Thanks.
